### ISSUE TITLE
Add usage examples to documentation, and help text for each sub-command.

### ISSUE THREAD

**Issue by ericgeldmacher** [ericgeldmacher/nbcli#16]

**Comment by awfki** [comment]
Yes, can we have examples, please.  I can't figure out how to select columns with the filter command.   When I use --cols it never returns anything.  

Oh!  I hadn't left it long enough.  I did `nbcli filter device --cols "name" SERIALNUMBER` and it returned EVERY device when I just want the one that matched.  

Ahh, `nbcli filter device SERIALNUMBER--cols "Name" "Serial"` is what I want.  The usage text caused me to think the option needed to be up front and then I was confused about multiple columns as it looked like they're just space separated.

Leaving this so you'll know examples are wanted, and maybe it'll help someone else.

**Comment by ericgeldmacher** [comment]
@awfki 
Improvements to the documentation have been slow due to an increase in my professional workload unrelated to Netbox. Continued work on the development and documentation of nbcli are still planned and a high priority for when I have time to work on it.
It is something I use almost every day for work and I am trying to get my coworkers to utilize it more as well.

**Comment by ericgeldmacher** [comment]
> Oh! I hadn't left it long enough. I did nbcli filter device --cols "name" SERIALNUMBER and it returned EVERY device when I just want the one that matched.

#32 Limit results from `nbcli filter` by default to prevent returning all objects when that behavior is not wanted.

**Comment by ericgeldmacher** [comment]
@awfki 
Some documentation updates have been pushed for the [filter](https://ericgeldmacher.github.io/nbcli/latest/commands/filter/) command, including details on the [--cols](https://ericgeldmacher.github.io/nbcli/latest/commands/filter/#-cols) flag.
I hope these are helpful to you.
This also includes a new dev release, with better help text, and some bug fixes for newer versions of Netbox.
So make sure to run:
```
pip3 install --upgrade nbcli
```
to get those fixes.
